src/kudu/consensus/log.cc

```diff
@@ -1270,7 +1270,7 @@ LogEntryBatch::~LogEntryBatch() {
     for (LogEntryPB& entry : *entry_batch_pb_->mutable_entry()) {
       // ReplicateMsg elements are owned by and must be freed by the caller
       // (e.g. the LogCache).
-      entry.release_replicate();
+      std::ignore = entry.release_replicate();
```

Comment:
Higher version of protobuf slaps a [[nodiscard]] on the release function, which errors out the compiler if the function result is unused.

The comment above states that here it's done on purpose, so we assign it to `std::ignore`. `std::ignore` is like the `/dev/null` of C++.